src/gui/modalMenu.cpp

```diff
@@ -155,7 +155,7 @@ bool GUIModalMenu::DoubleClickDetection(const SEvent &event)
 			return false;
 		}
 
-		SEvent translated = SEvent();
+		SEvent translated;
```

Comment:
```suggestion
		SEvent translated{};
```

Reply:
I'm not fond of Irrlicht, why braces are exactly needed? (I'd like to understand something before changing it)